lib/xcflushd/threading.rb

```diff
@@ -3,10 +3,8 @@
 
 module Xcflushd
   module Threading
-    def self.default_threads_value
-      cpus = Concurrent.processor_count
-      # default thread pool minimum is zero
-      return 0, cpus * 4
+    def self.default_threads
+      Concurrent.processor_count
```

Comment:
This has been changed to have an effective value of `processor_count` threads instead of 4 times that value. Why? Maybe a bit of testing is needed to see that it really makes no difference, but I'd lean on defaulting to 4 times, since the process is I/O bound and threads are likely to block.